client/components/Modals/PhotoModal.jsx

```diff
@@ -45,8 +45,9 @@ let Photo = styled.div`
   }
 `;
 const Button = styled.button`
-#photomodal & [
-]`;
+  #photo-modal & {
+  }
+`;
 
```

Comment:
looks good!  does this cover the last of your modals?

Reply:
What do you mean the last of my modals?